shop-be/import-service/serverless.ts

```diff
@@ -0,0 +1,91 @@
+import type { AWS } from '@serverless/typescript';
+
+import importProductsFile from '@functions/importProductsFile';
+import importFileParser from '@functions/importFileParser';
+const BUCKET_NAME = 'marusel-bucket-aws';
+
+const serverlessConfiguration: AWS = {
+  service: 'import-service',
+  frameworkVersion: '2',
+  plugins: ['serverless-esbuild'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs14.x',
+    region: 'eu-west-1',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      BUCKET_NAME: 'marusel-bucket-aws',
+      NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+    },
+    lambdaHashingVersion: '20201221',
+    iamRoleStatements: [
+      {
+        Effect: 'Allow',
+        Action: ['s3:ListBucket'],
+        Resource: "arn:aws:s3:::marusel-bucket-aws"
+      },
+      {
+        Effect: 'Allow',
+        Action: ['s3:*'],
```

Comment:
Good practice to apply a more specific permissions - because now you allow to do everything with the object

Reply:
https://docs.aws.amazon.com/AmazonS3/latest/API/API_Operations.html